### DISCUSSION TITLE
Add option for `IMMcustom` and if possible `SDMcustom`

### DISCUSSION THREAD

**Discussion by GidonFrischkorn** [venpopov/bmm#183] [Ideas]
Depending on the complexity of the `M3custom`, I thought about implementing an `IMMcustom` that allows users to specify their custom activation formulas for different categories and adjust the formulae for the mixture weights accordingly. I have to think a little about how to deal with identifying different categories of swaps (e.g., swaps from WM as implemented currently, and for example swaps from LTM if there is semantic or pre-learned knowledge). 

Depending on the complexity of the `SDMfull` it would be great if there is a possibility to also have an `SDMcustom` option. But like the `IMMcustom` this is definitely something for future releases and not from the initial CRAN release.

**Comment by venpopov** [comment]
Closing this as we now use the "version" argument #178
